v3/plugins/joaompinto/asciidoctor-vscode/latest/meta.yaml

```diff
@@ -0,0 +1,12 @@
+publisher: joaompinto
+apiVersion: v2
+id: joaompinto.asciidoctor-vscode
+version: latest
+type: VS Code extension
+name: asciidoctor-vscode
```

Comment:
The other vscode plugins have name formatted `vscode-<name>`. Could we update this to be `vscode-asciidoctor`?

Reply:
ok. Done